lib/matplotlib/tests/test_ticker.py

```diff
@@ -511,35 +511,66 @@ def test_formatstrformatter():
     assert '002-01' == tmp_form(2, 1)
 
 
-percentformatter_test_cases = (
```

Comment:
#7993 please. I'd like to rebase all my open PRs onto it. It's ready to go, just waiting for Mac test to run.